tests/testing_common.py

```diff
@@ -1419,6 +1419,9 @@ def _test_weighted_combination_of_adapters(self, model_id, config_cls, config_kw
         if issubclass(config_cls, AdaLoraConfig):
             # AdaLora does not support adding more than 1 adapter
             return pytest.skip(f"Test not applicable for {config_cls}")
+        if model_id.endswith("qwen2"):
```

Comment:
Is `model_id` a sufficiently reliable way to detect if the model is `qwen2` type? Could we not look into its config and infer it from there?

Reply:
Make sense. I will respect your judgement then.